server/mdm/nanomdm/storage/mysql/mysql.go

```diff
@@ -346,3 +346,8 @@ func (s *MySQLStorage) updateLastSeenBatch(ctx context.Context, ids []string) {
 		s.logger.Info("msg", "error batch updating nano_enrollments.last_seen_at", "err", err)
 	}
 }
+
+func (s *MySQLStorage) ExpandEmbeddedSecrets(_ context.Context, document string) (string, error) {
+	s.logger.Info("level", "error", "err", "MySQLStorage.ExpandEmbeddedSecrets not implemented")
```

Comment:
I'm not super familiar with NanoMDM, is there a reason we only implement this on the MultiAllStorage and not the MySQLStorage?

Reply:
I'm not an expert either, but it looks like MultiAllStorage is just a pass-through. Maybe so you can have a mix of storage mediums -- some things in DB and some things in files, for example.